packages/fetchai/connections/p2p_libp2p_client/connection.py

```diff
@@ -260,7 +260,7 @@ async def _process_messages(self) -> None:
             assert self._in_queue is not None, "Input queue not initialized."
             self._in_queue.put_nowait(data)
             if data is None:
-                break
+                break  # pragma: no cover
```

Comment:
on my local machine, with file (missing from develop) `test_errors.py`, this is already covered by  `tests/test_packages/test_connections/test_p2p_libp2p_client/test_errors.py::TestLibp2pClientConnectionNodeDisconnected::test_node_disconnected`
(https://github.com/fetchai/agents-aea/blob/feature/libp2p-improve-coverage/tests/test_packages/test_connections/test_p2p_libp2p_client/test_errors.py#L128)